common/persistence/src/main/java/app/coronawarn/datadonation/common/persistence/domain/metrics/TestResultMetadata.java

```diff
@@ -2,13 +2,17 @@
 
 import app.coronawarn.datadonation.common.persistence.domain.metrics.embeddable.UserMetadataDetails;
 import javax.validation.constraints.NotNull;
+import org.hibernate.validator.constraints.Range;
 import org.springframework.data.relational.core.mapping.Embedded;
 import org.springframework.data.relational.core.mapping.Embedded.OnEmpty;
 
 public class TestResultMetadata extends DataDonationMetric {
-  
+
+  private static final long MIN_RISK_LEVEL = 0;
+  private static final long MAX_RISK_LEVEL = 3;
```

Comment:
shouldn't this be adjusted to 7 or 8?

```suggestion
  private static final long MAX_RISK_LEVEL = 8;
```

Reply:
Based on the tech spec for metadata tables this should be from 0 - 3.